### PR TITLE
WE-909 implement getObjectsIdOnly

### PR DESCRIPTION
**Pull request by janburak**
[//]: # (Request Template Witsml Explorer)

[//]: # (Thank you for contributing to WITSML Explorer! Before submitting this PR, please fill in the following template.)

## Fixes

[//]: # (Write the GitHub issue number starting with #, or, for Equinor only, the Jira issue number starting with WE-)

This pull request fixes WE-909

## Description

[//]: # (Please include a written summary of the changes and which issue is fixed. List any dependencies that are required for this change._)

* Implement API route to get objects of any type using IdOnly. It can be used to efficiently check existence of objects.
* Change ObjectOnWellbore in API from abstract class to class. If ObjectOnWellbore needs to be abstract again for any reason then one can make a new class that represents only the return value of GetObjectsIdOnly.

## Type of change

[//]: # (Mark any of the types of change that apply.)

* [ ] Bugfix
* [x] New feature (non-breaking change which adds functionality)
* [ ] Enhancement of existing functionality
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ] This change requires a documentation update

## Impacted Areas in Application

[//]: # (List general components of the application that this PR will affect)

* [x] Frontend
* [x] API
* [ ] WITSML
* [ ] Other (please describe)

## Checklist:

[//]: # (Please tick all the boxes or remove the ones that aren't needed and explain why)

*Communication*
* [ ] I have made corresponding changes to the documentation
* [ ] PR affects application security

*Code quality*
* [x] I have self-reviewed my code
* [x] No new warnings are generated

*Test coverage*
* [x] New code is covered by passing tests

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: Src/WitsmlExplorer.Api/HttpHandlers/ObjectHandler.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+using WitsmlExplorer.Api.Models;
+using WitsmlExplorer.Api.Services;
+
+namespace WitsmlExplorer.Api.HttpHandlers
+{
+    public static class ObjectHandler
+    {
+        [Produces(typeof(ObjectOnWellbore[]))]
+        public static async Task<IResult> GetObjectsIdOnly(string wellUid, string wellboreUid, EntityType objectType, IObjectService objectService)
```
Comment (HakonHesselberg) [inline Src/WitsmlExplorer.Api/HttpHandlers/ObjectHandler.cs:14]: `GetObjectsIdOnly` seems ambigious. Is it possible to use a more clear name?

File: Tests/WitsmlExplorer.Api.Tests/Services/ObjectServiceTests.cs [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                        queryIn.Objects.First().Uid == "" &&
+                        queryIn.TypeName == new WitsmlMessages().TypeName
+                    ),
+                    It.Is<OptionsIn>((ops) => ops.ReturnElements == ReturnElements.IdOnly)))
+                .ReturnsAsync(messages);
+
+            IEnumerable<ObjectOnWellbore> result = await _service.GetObjectsIdOnly(uidWell, uidWellbore, EntityType.Message);
+            Assert.Empty(result);
+        }
+
```
Comment (HakonHesselberg) [inline Tests/WitsmlExplorer.Api.Tests/Services/ObjectServiceTests.cs:120]: Empty line

**Review by HakonHesselberg — APPROVED** [review]
🧙

File: Src/WitsmlExplorer.Api/HttpHandlers/ObjectHandler.cs (hunk 1 above)
Comment (janburak) [reply Src/WitsmlExplorer.Api/HttpHandlers/ObjectHandler.cs:14]: It fits with the rest of the repo when talking about Object(OnWellbore). I wish we had a better name for it tho.

**Pull request merged**